lib/transforms.js

```diff
@@ -493,9 +493,45 @@ var versions = {
 
       return convertSlug(results);
 
-    }
+    },
+
+    "0.5.5": function(ans) {
+      // Update the version.
+      ans = version_incrementer("0.5.6")(ans);
 
+      // Convert 0.5.5 to 0.5.5
+      var convert = function convert(ans) {
+        if(!_.isObject(ans)) {
+          return ans;
+        }
+
+        return _.transform(ans, function(result, value, key, object) {
+          if ( key === "taxonomy" ) {
+            // Traverse into taxonomy
+            result[key] = convert(value);
+          } else if ( key === "sites" ) {
+            // Update all sites.  Add 'primary' attribute.
+            if (value.length && value.length > 0) {
+              var sites = value.map(function (site) {
+                site.primary = false;
+                return site;
+              });
 
+              result[key] = sites;
+            }
+          } else if ( key === "owner") {
+            // Update owner.  Add 'sponsored' attribute.
+            value.sponsored = false;
+            result[key] = value;
+          } else {
+            result[key] = value;
```

Comment:
EDIT: Probably something more like `if (key === 'additional_properties') { result[key] = value; } else { result[key] = convert(value); }` so as not to delete additional_properties